website/src/scss/sections/project.scss

```diff
@@ -0,0 +1,100 @@
+@import '../parts/utils/fonts';
+@import '../parts/utils/colors';
```

Comment:
You should never need to do this specifically in each file. Ideally you have all your imports in a single file (and then you import parts and other helpers) at the top and all your project-specific styles at the bottom (see frontend-libs / boilerplate)